modules/SampleModule/inc/samplemodule.hpp

```diff
@@ -4,28 +4,51 @@
 #include "module.hpp"
 #include "server.hpp"
 
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+#pragma once
+
+// Module specific publisher/subscriber
+namespace ROSChannels {
+    namespace SmOnInitResponse {
```

Comment:
Can you change this name, it's not clear what `SmOnInitResponse` means at a first glance.